src/test/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceServiceSpec.scala

```diff
@@ -843,6 +958,90 @@ class ResourceServiceSpec extends AnyFlatSpec with Matchers with ScalaFutures wi
     runAndWait(service.removeSubjectFromPolicy(FullyQualifiedPolicyId(resource, policyName), otherUserInfo.userId, samRequestContext))
   }
 
+  "addSubjectToPolicy" should "call CloudExtensions.onGroupUpdate when member added" in {
+    val mockCloudExtensions: CloudExtensions = mock[CloudExtensions](RETURNS_SMART_NULLS)
+    val mockDirectoryDAO: DirectoryDAO = mock[DirectoryDAO](RETURNS_SMART_NULLS)
+    val resourceService = new ResourceService(Map.empty,
+      mock[PolicyEvaluatorService](RETURNS_SMART_NULLS),
+      mock[AccessPolicyDAO](RETURNS_SMART_NULLS),
+      mockDirectoryDAO,
+      mockCloudExtensions,
+      "")
+
+    val policyId = FullyQualifiedPolicyId(FullyQualifiedResourceId(defaultResourceType.name, ResourceId("testR")), AccessPolicyName("testA"))
```

Comment:
TOL: I noticed `testR`, `testA`, and `testU` are used a lot across these tests, maybe it makes sense to make a variable or name them higher up? Unless we're testing on responses I'm not too worried, but I could see that popping up.

Reply:
the values are irrelevant for the tests themselves and I err on the side of keeping tests self contained